dist/Text-Abbrev/t/Abbrev.t

```diff
@@ -7,7 +7,7 @@ use Text::Abbrev;
 print "ok 1\n";
 
 # old style as reference
-local(%x);
+our %x;
```

Comment:
Why is this `our` instead of `my`?

Reply:
Because with `my %x` we get this:
```
$ cd t; ./perl harness -v ../dist/Text-Abbrev/t/Abbrev.t; cd -
Name "main::x" used only once: possible typo at t/Abbrev.t line 13.
Use of uninitialized value $x{"list"} in string eq at t/Abbrev.t line 22.
Use of uninitialized value $x{"a"} in string eq at t/Abbrev.t line 29.

...
```
So I'm leaving it as `our`.